test/AWS.Deploy.CLI.Common.UnitTests/Recipes/Validation/ECSFargateOptionSettingItemValidationTests.cs

```diff
@@ -6,12 +6,21 @@
 using AWS.Deploy.CLI.Common.UnitTests.IO;
 using AWS.Deploy.Common;
 using AWS.Deploy.Common.IO;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Amazon.CloudControlApi.Model;
+using Amazon.ECS.Model;
+using AWS.Deploy.Common;
```

Comment:
Can you clean up the build warnings on the duplicate `usings` here - mostly because they seem to then reappear on every PR.

Similar comment down on lines 206 and 232, though I can't comment on it.

Reply:
Done